go/tasks/plugins/k8s/spark/spark_test.go

```diff
@@ -47,6 +47,7 @@ var (
 		"spark.flyte.feature1.enabled":    "true",
 		"spark.flyteorg.feature2.enabled": "true",
 		"spark.flyteorg.feature3.enabled": "true",
+		"spark.batchScheduler":            "volcano",
```

Comment:
why add it to spark-conf? Can we add it to just regular config? And I would prefer the config to say
scheduler: `default` - as default
and users can then suplement it as `scheduler`: `volcano`

Reply:
cc @akhurana001

Reply:
Reason for spark conf is that for our use case we don't want to enable it for everyone, we'd like to try it out with a specific workflow. Adding it to regular config will set it for all unless we move to a config for each project/workflow. And I have no preference on whether `batchScheduler` or `scheduler`. Spark uses batch scheduler so figured I'd keep the same naming convention